routes/blogy.js

```diff
@@ -1,40 +1,39 @@
 const router = require("express").Router();
-const blog = require("../backend_model/blog_post");
-const blogcomment = require("../backend_model/Comment_post");
+const blog = require("../model/post");
```

Comment:
Change `blog` to `Blog`. 

The uppercase just makes it more descriptive that `Blog` is a class/like a class